src/App.jsx

```diff
@@ -1,3 +1,95 @@
-export default function App() {
-  return <h1 className="text-3xl font-bold underline bg-blue-500">React exercises</h1>;
-}
+import { useState, useEffect } from "react";
+import Dice from "./Components/Dice";
+import { nanoid } from "nanoid";
+import Confetti from "react-confetti";
+
+const App = () => {
+  const [dice, setDice] = useState(generateNewDice);
+  const [tenzies, setTenzies] = useState(false);
+
+  function generateNewDice() {
+    const newDice = [];
+    for (let i = 0; i < 10; i++) {
+      newDice.push({
+        id: nanoid(),
+        value: Math.ceil(Math.random() * 6),
+        isHeld: false,
+      });
+    }
+    return newDice;
+  }
+
+  const rollNewDice = () => {
+    if (!tenzies) {
+      setDice(
+        dice.map((die) => {
```

Comment:
As long as you have only a single argument, there is no need to put it into the brackets, because by default code format removes them, so you should also remove them, 'cause it even improves the readability of the codes.